openbb_terminal/common/prediction_techniques/expo_view.py

```diff
@@ -25,6 +25,17 @@
 # pylint: disable=too-many-arguments
 
 
+def dt_format(x):
```

Comment:
@martinb-bb would this mess up timestamps in intraday data

Reply:
We are only doing daily for now. 
intraday will be added in future to the forecasting menu.

Reply:
ah, ok, we have a safeguard there

Reply:
Yes 😄 , we need actual frequency to the timeseries. Users cannot do forecasting on daily and then provide intra-day. They must be 2 separate requests.